src/components/Project/Project.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { mobileProject, mobileProjectTitle } from './Project.module.scss';
+
+export const Project = ({ link, description, title }) => {
+  return (
+    <>
+      <h1 className={`${mobileProjectTitle}`}>{title}</h1>
+      <div className={`${mobileProject}`}>
+        <div>{description}</div>
+        <a href={link}>Zobacz nasz projekt w praktyce {/* ARROW */} &#10140;</a>
```

Comment:
can you extract this encoded arrow into variable and then interpolate it in this element?
this way extra comment wont be needed